Tools/upload_wiki_icons.py

```
@@ -0,0 +1,131 @@
+
+def graphql(base_url: str, token: str, query: str) -> dict:
+    body = json.dumps({"query": query}).encode("utf-8")
+    request = urllib.request.Request(
+        f"{base_url}/graphql",
+        data=body,
+        headers={
+            "Authorization": f"Bearer {token}",
+            "Content-Type": "application/json",
+        },
+        method="POST",
+    )
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)